lib/application/widgets/theme/custom_button.dart

```diff
@@ -292,7 +298,11 @@ class CustomButtonState extends State<CustomButton> {
           state = isPressed ? ButtonState.pressed : ButtonState.normal;
         });
       },
-      child: Center(child: Row(mainAxisAlignment: MainAxisAlignment.center, children: rowWidgets)),
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.center,
+        mainAxisSize: widget.shrink ? MainAxisSize.min : MainAxisSize.max,
+        children: [context.horizontalBox(Spacing.small), ...rowWidgets, context.horizontalBox(Spacing.small)],
```

Comment:
Actually based on the tests made here https://github.com/olmps/memo/pull/229#issuecomment-964196783 I would even remove this paddings, since they should be defined by the buttons parent if necessary (since we've scenarios where we wouldn't like to have any implicit padding)